docs_app/content/guide/testing/marble-testing.md

```diff
@@ -278,14 +272,14 @@ On a related note, you also can't currently assert delays of zero, even with `As
 
 ### Behavior is different outside of `testScheduler.run(callback)`
 
-The `TestScheduler` has been around since v5, but was actually intended for testing RxJS itself by the maintainers, rather than for use in regular user apps. Because of this, some of the default behaviors and features of the TestScheduler did not work well (or at all) for users. In v6 we introduced the `testScheduler.run(callback)` method which allowed us to provide new defaults and features in a non-breaking way, but it's still possible to [use the TestScheduler outside](./guide/testing/internal-marble-tests) of `testScheduler.run(callback)`. It's important to note that if you do so, there are some major differences in how it will behave.
+The `TestScheduler` has been around since v5, but was actually intended for testing RxJS itself by the maintainers, rather than for use in regular user apps. Because of this, some of the default behaviors and features of the TestScheduler did not work well (or at all) for users. In v6 we introduced the `testScheduler.run(callback)` method which allowed us to provide new defaults and features in a non-breaking way, but it's still possible to use the TestScheduler outside of `testScheduler.run(callback)`. It's important to note that if you do so, there are some major differences in how it will behave.
```

Comment:
I think we should keep the link to the old documentation, but change it to a blob-specific URL - https://github.com/ReactiveX/rxjs/blob/7113ae4b451dd8463fae71b68edab96079d089df/docs_app/content/guide/testing/internal-marble-tests.md - so that the file can still be deleted in this PR.